frontend/src/metabase/visualizations/echarts/cartesian/option/index.ts

```diff
@@ -54,7 +54,6 @@ export const getCartesianChartOption = (
   ];
 
   return {
-    animation: false,
```

Comment:
Why remove this?

Reply:
If we disable animation globally, hover color transitions are immediate and do not look good